MD_CONF_DEBUGGING changes CFLAGS and CXXFLAGS with a horrible magic so disable
it to prevent compilation errors.

downloaded from
http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/dev-libs/libhid/files/libhid-0.2.16-gentoo.patch?revision=1.1&view=markup

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

diff -uNr libhid-0.2.16.org/configure.ac libhid-0.2.16/configure.ac
--- libhid-0.2.16.org/configure.ac	2007-04-01 22:32:10.000000000 +0200
+++ libhid-0.2.16/configure.ac	2014-10-19 21:31:00.550091947 +0200
@@ -78,7 +78,6 @@
 AC_C_BIGENDIAN
 AC_C_STRINGIZE
 
-MD_CONF_DEBUGGING
 MD_CONF_COMPILER
 AM_CONDITIONAL(OS_LINUX, test "$MD_OS" = "linux")
 AM_CONDITIONAL(OS_BSD, test "$MD_OS" = "bsd")
